rfcs/text/0024/ticket.yml

```diff
@@ -0,0 +1,151 @@
+---
+- name: ticket
+  group: 2
+  title: Ticket
+  short: Fields describing details of a ticket.
+  description: >
+    A ticket is defined as a documented record of any form of request or reported issue.
+    Tickets can be associated with hardware, software, access, procurement, etc. requests.
+    They can also be associated with ITIL /ITSM support processes
+    Ticket fields provide key basic details about the ticket itself.
+  footnote: >
+    Ticket types include, but are not limited to:
+    Alarm, Event, Incident, Problem, Case, Bug, Feature Request, Task, Epic, User Story,
+    Knowledge Article, Change Request, etc.
+  type: group
+
+  fields:
+
+    - name: assignee
+      level: extended
+      type: keyword
+      short: Assigned to name
+      description: >
+        Name of the person or group with whom the ticket is currently assigned.
+      example: Sector_7-G_Safety_Desk
+
+    - name: created
+      level: core
+      type: date
+      short: Timestamp the ticket was created
+      description: >
+        Timestamp the ticket was created.
+      example: 2021-03-09T17:05:23.773Z
+
+    - name: description.long
+      level: core
+      type: text
+      short: Long description of the ticket
+      description: >
+        Long description / body of the ticket.  Typically contains all details and
+        specifics of the ticket.
+      example: >
+        A fiber cut has occurred in Springfield due to a down telephone pole.  All internet and
+        IP related services are down diddly offline. Customer call in-rates are elevated.
+
+    - name: description.short
+      level: core
+      type: text
+      short: Short description of the ticket
+      description: >
+        Short description / title of the ticket.  Typically, a brief overview.
+      example: >
+        Fiber cut in Springfield - internet service down
+
+    - name: details
+      level: extended
+      type: object
+      object_type: keyword
+      example: "{\"affected_version\": \"1.2.3\", \"env\": \"production\", \"resolution\": \"unresolved\", \"service\": \"Internet\"}"
+      short: Custom key/value pairs.
+      description: >
+        Custom key/value pairs.
+        Can be used to add meta information about the ticket, or include additional customized
+        ticket fields. Should not contain nested objects.
+        All values are stored as keyword.
+
+    - name: number
```

Comment:
I thought about `.id` as well as it aligns better with Elastic concepts.  However, I stuck with `.number` because that is the common name used in ticketing platforms and thought it may be easier for folks to grok that way.  I could go either way with this; figured it was best to throw something out there and let the community weigh in to get a general consensus of `.id` vs. `.number`.

Reply:
I agree; even when some systems use alphanumeric conventions like CHG12345, INC12345, TEST-12345, etc., the term "ticket number" is generally understood. 

I still lean towards using a generic convention like `id` to avoid potential confusion of a `number` field containing both letters and numerals.